templates/security/notices.html

```diff
@@ -1 +1,142 @@
+          </div>
+          <div class="col-2" style="display: flex; align-items: flex-end;">
+            <button type="submit" class="p-button--positive js-submit">Search{% if search %} again{% endif %}</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </form>
+</section>
+<section class="p-strip">
+  {% if search %}
+  <div class="row">
+    <div class="col-8">
+      <h2 class="p-heading--four">
+        {{ notices|length }} of {{ pagination.total_results }} results
+      </h2>
+    </div>
+    <div class="col-4 p-form p-form--inline">
+      <div class="p-form__group">
+        <label class="p-form__label">Sort by:</label>
+        <select class="p-form__control u-no-margin--bottom js-order-select">
+          <option value="newest" {% if request.args.get('order') == 'newest'%} selected {% endif %}>Newest</option>
+          <option value="oldest" {% if request.args.get('order') == 'oldest'%} selected {% endif %}>Oldest</option>
+        </select>
+      </div>
+    </div>
+  </div>
+  <div class="u-fixed-width" style="margin-bottom: 2rem; margin-top: 2rem;">
+    <hr class="u-hide--small"/>
+  </div>
+  {% endif %}
+  <div class="row">
+    <div class="col-9">
+      {% if not search %}
+      <h2>Latest Notices</h2>
+      {% endif %}
+      {% if notices %}
+      {% for notice in notices %}
+      {% include "security/_notice-brief.html" %}
+      {% endfor %}
+      {% else %}
+      <h2>No notices found...</h2>
+      {% endif %}
+    </div>
+    {% if not search %}
+    <div class="col-3 u-hide--small">
+      {% include "security/_subscribe.html" %}
+    </div>
+    {% endif %}
+  </div>
+  {% with total_pages = pagination.total_pages, current_page=pagination.current_page %}
+  {% include "shared/_pagination.html" %}
+  {% endwith %}
+  {% if not search %}
+  <div class="row u-hide--medium u-hide--large">
+    <div class="col-12">
+      {% include "security/_subscribe.html" %}
+    </div>
+  </div>
+  {% endif %}
+</section>
+{% with first_item="_security_discussion", second_item="_security_esm", third_item="_security_further_reading" %}
+{% include "shared/contextual_footers/_contextual_footer.html" %}
+{% endwith %}
+<script>
+  var form = document.querySelector(".js-form");
+  var orderInput = document.querySelector(".js-order-input");
+  var orderSelect = document.querySelector(".js-order-select");
+  var details = document.querySelector(".js-details");
+  var release = document.querySelector(".js-release");
+  var submit = document.querySelector(".js-submit");
+
+  // Submit form on order change 
+  if (orderSelect != null) {
+    orderSelect.onchange = function(event) {
+      orderInput.value = event.target.value;
+      form.submit();
+    }
+  }
+
+  // Disable search when form is empty
+  function toogleSearch() {
+    submit.disabled = !details.value && !release.value;
+  }
+
+  form.onchange = function(event) {
+    toogleSearch();
+  }
+
```

Comment:
Sorry, one more suggestion - first thing I did was type something into the `#details` input, but the submit button didn't activate until I'd clicked somewhere else on the page. You could also attach a `keyup` event:

```suggestion

  details.onkeyup = function() {
    toogleSearch();
  }
  
```

Reply:
Added